cmd/deploy/remote.go

```diff
@@ -62,7 +62,7 @@ WORKDIR /okteto/src
 ENV OKTETO_INVALIDATE_CACHE {{ .RandomInt }}
 RUN okteto deploy --log-output=json {{ .DeployFlags }}
 `
-	dockerignoreName = "deploy.dockerignore"
+	dockerignoreName = ".oktetodeployignore"
```

Comment:
Should we change also the value here? https://github.com/okteto/okteto/blob/adrian/deploy-destroy-same-way/cmd/destroy/remote.go#L52

Reply:
oh yes, I forgot the change for destroy in remote! Thanks @ifbyol